pkgs/top-level/python-packages.nix

```diff
@@ -22252,6 +22252,27 @@ in {
   };
 
 
+  rst2html5 = buildPythonPackage rec {
+    name = "rst2html5-${version}";
+    version = "1.8.2";
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/bc/33/86da22a1b2fc63403f19567d9e07be311881abeb3e0380d7f6d7faf8b8b6/${name}.tar.gz";
```

Comment:
One that always works is 
```
url = "mirror://pypi/${builtins.substring 0 1 pname}/${pname}/${name}.tar.gz";
```
where `pname` is the name of the package on PyPI, which should be the same as the name of the package, so, `rst2html5`
```
pname = "rst2html5";
version = "1.8.2";
name = "${pname}-${version}";